.pre-commit-config.yaml

```diff
@@ -1,12 +1,17 @@
+ci:
+  autoupdate_schedule: "monthly"  # Like dependabot
+  autoupdate_commit_msg: "chore: update pre-commit hooks"
+  autofix_prs: false  # Comment "pre-commit.ci autofix" on a PR to trigger
+
```

Comment:
I've not used pre-commit.ci before. I'm assuming this is correct and you'll handle whatever signup / authentication is needed to hook this repo into pre-commit.ci

Reply:
Yes, need to flip a switch on my other GitHub account. It will be just a minute...